scripts/wmata_data/distcalc.py

```diff
@@ -0,0 +1,149 @@
+import sys
+import csv
+import requests
+
+#TODO replace these with code4dc keys
+WMATA_HEADERS = {'api_key': ''}
+MAPBOX_PARAMS = {'access_token': ''}
+
```

Comment:
I wasn't sure where this script should go so rather than hardcoding the secret.json path in I just added it as an arg to the script.  For now.  If you know where dist.py is going, I can hardcode it although wasn't sure if that is what you all are doing in other scripts.